nautobot_chatops/choices.py

```diff
@@ -41,6 +41,6 @@ class CommandTokenPlatformChoices(ChoiceSet):
     MATTERMOST = "mattermost"
     SLACK = "slack"
     MS_TEAMS = "microsoft_teams"
-    WEBEX_TEAMS = "webex_teams"
+    WEBEX = "webex"
```

Comment:
Should we keep WEBEX_TEAMS for this particular setup?

Reply:
Currently only Mattermost is being used, so this is just forward looking.
As for keeping it, I'm not sure why we'd want to when everything else is changing to just WEBEX.  Is there any specific scenario you had in mind for keeping it as WEBEX_TEAMS?

Reply:
Webex is good hear, that is correct it is not used in code yet, but I have plans for it.